core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html

```diff
@@ -78,4 +78,31 @@
 # once a day on the 1st and 15th of every month except December
 H H 1,15 1-11 *
 </pre>
+<p>
+  <b>Timezone specification</b>. <br/>
+  Periodic tasks are normally executed at the scheduled time in the timezone of 
+  the Jenkins master JVM (currently Central European Time). 
```

Comment:
I still think a Jelly based help file could at least show the current time zone.